docs/authenticate.md

```diff
@@ -30,40 +32,61 @@ To authenticate to the HERE platform and start working with HERE Data SDK for C+
 
 You get an access token.
 
-You can use the `AuthenticationSettings` object to create the `OlpClientSettings` object. For more information, see the [related](https://developer.here.com/documentation/sdk-cpp/dev_guide/topics/create-olp-client-settings.html) section in the Developer Guide.
+You can use the `AuthenticationSettings` object to create the `OlpClientSettings` object. For more information, see the [related section](https://developer.here.com/documentation/sdk-cpp/dev_guide/topics/create-olp-client-settings.html) in the Developer Guide.
 
 ## Authenticate using project authentication
 
-1. Create your application and get your API key.
+1. Get your platform credentials.
 
-   For instructions, see the [Manage Apps](https://developer.here.com/documentation/access-control/user_guide/topics/manage-apps.html) section in the Terms and Permissions User Guide.
+   For instructions, see the [Register Your Application](https://developer.here.com/documentation/identity-access-management/dev_guide/topics/plat-token.html#step-1-register-your-application) section in the Identity & Access Management Developer Guide.
 
-2. Create a function that returns your API key.
-3. Set up the `AuthenticationSettings` object with your function that returns the API key.
+2. Initialize the `AuthenticationCredentials` class using the **here.access.key.іd** and **here.access.key.secret** from the `credentials.properties` file as `kKeyId` and `kKeySecret` respectively.
 
-   > Note: Do not trigger any tasks on `TaskScheduler` as this might result in a deadlock.
+   > Note: You can also retrieve your credentials from the `credentials.properties` file using the `ReadFromFile` method. For more information, see the [related API documentation](https://developer.here.com/documentation/sdk-cpp/api_reference/classolp_1_1authentication_1_1_authentication_credentials.html#a6bfd8347ebe89e45713b966e621dccdd).
 
    ```cpp
-   olp::client::AuthenticationSettings auth_settings;
-   auth_settings.api_key_provider =
-   [](){ return "your-api-key"; };
+   olp::authentication::AuthenticationCredentials credentials(kKeyId, kKeySecret);
+   ```
+
+3. Create an authentication client.
+
+   ```cpp
+   olp::authentication::AuthenticationSettings settings;
+   settings.task_scheduler = task_scheduler;
+   settings.network_request_handler = http_client;
+   authentication::AutnhentucationClient client(settings);
+   ```
+
+4. Create the `SignInProperties` object with your project HRN.
+
+   ```cpp
+   authentication::SignInProperties signin_properties;
+   signin_properties.scope = "<project hrn>";
```

Comment:
<project hrn> is this correct? Is the project a HRN like stricture or just a normal ID?

Reply:
Changed to ID.